src/syntax.ml

```diff
@@ -153,21 +153,17 @@ type p_config =
   | P_config_binop   of binop
   (** Define (or redefine) a binary operator (e.g., ["+"] or ["×"]). *)
 
-type require_mode =
-  | P_require_default
-  (** Just require the module. *)
-  | P_require_open
-  (** Require the module and open it. *)
-  | P_require_as of ident
-  (** Require the module and aliases it with the given indentifier. *)
-
 (** Parser-level representation of a single command. *)
 type p_statement = (ident * p_arg list * p_type) loc
 
 type p_command_aux =
-  | P_require    of module_path * require_mode
```

Comment:
You should use a boolean to say whether the modules must be opened. That would save a constructor.